### DISCUSSION TITLE
Changing Playback Rate was more usable before

### DISCUSSION THREAD

**Discussion by ahrl9** [ppy/osu#28987] [Ideas] [answered]
### Type

Game behaviour

### Bug description

When using slower playback rates in the editor, it's known that the sound gets off sync, but it also looses a lot more quality compared to stable. I think maintaining pitch in some cases is pretty important, but before the pitch was the same and it pitched down when adjusting the playback speed, the sound was just way more accurate and you could distinguish drums or percussions way easier. Now it feels like there is a delay effect on everything, all sounds get played like 50 times, it's just not usable like this. Using the waveform is also not possible in many cases, because in genres like EDM or metal you can't differentiate anything through the wavetable because everything is mixed for loudness, it just looks like a fat block in the editor.

Problem: Can't use slower playback rates, due to indistinguishable sounds from quality loss and unsynchronized playback

Fix Suggestions:
- use a different approach of slowing down the music (the stable one worked just fine)
- keep the option to use the pitching down algorithm for slowing down (you can make it an editor setting)
- calculate an accurate offset for the slower song to not be off-sync

### Screenshots or videos

A screenshot example for unusable waveform:
![image](https://github.com/user-attachments/assets/d0fb4796-1802-4545-9dcd-8c744198aaf8)

### Version

2024.718.1-lazer

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/16319893/compressed-logs.zip)

**Comment by XeZey** [comment]
personally I never took issue with having lower pitched music while editing anyway and probably would even prefer it over same pitch if I could choose, though I might be in the minority there

**Comment by bdach** [comment, marked as answer]
should be resolved with https://github.com/ppy/osu-framework/pull/6354

**Reply by Freeze7x** [reply]
This solves the unsynced playback, but what about the audio quality? Could we maybe consider it being a toggle so we can have the best of both worlds?

**Reply by bdach** [reply]
Audio quality matches stable now. I don't foresee toggles for pitch unless there is significant support.
